docs/faq.md

```diff
@@ -258,8 +258,8 @@ one to expose DNS to the internet.
 
 To do this with ExternalDNS you can use the `--annotation-filter` to specifically tie an instance of ExternalDNS to
 an instance of a ingress controller. Let's assume you have two ingress controllers `nginx-internal` and `nginx-external`
-then you can start two ExternalDNS providers one with `--annotation-filter=kubernetes.io/ingress.class=nginx-internal`
-and one with `--annotation-filter=kubernetes.io/ingress.class=nginx-external`.
+then you can start two ExternalDNS providers one with `--annotation-filter=kubernetes.io/ingress.class in (nginx-internal)`
+and one with `--annotation-filter=kubernetes.io/ingress.class in (nginx-external)`.
```

Comment:
We must say that the flan needs to be quoted otherwise it will not work.

```
external-dns --annotation-filter=kubernetes.io/ingress.class in (nginx-external) --source=ingress --provider=aws
zsh: missing delimiter for 'g' glob qualifier
```

👇  works instead
```
external-dns --annotation-filter="kubernetes.io/ingress.class in (nginx-external)" --source=ingress --provider=aws
```